pyfr/solvers/acnavstokes/elements.py

```diff
@@ -6,6 +6,13 @@
 
 class ACNavierStokesElements(BaseACFluidElements,
                              BaseAdvectionDiffusionElements):
+    gradconvarmap = {k : [f'grad_{var}' for var in v] for k, v in BaseACFluidElements.convarmap.items()}
```

Comment:
Unsure if we need these maps; they can always be synthesised from convarmap and privarmap.

Reply:
True, I'll then check if the gradient to be computed is the gradient of a primitive variable or not